src/gpodder/gtkui/desktop/preferences.py

```diff
@@ -321,21 +321,37 @@ def new(self):
 
         gpodder.user_extensions.on_ui_object_available('preferences-gtk', self)
 
+        self.inject_extensions_preferences(init=True)
+
+        self.prefs_stack.foreach(self._wrap_checkbox_labels)
+
+    def _wrap_checkbox_labels(self, w, *args):
+        if w.get_name().startswith("no_label_wrap"):
+            return
+        elif isinstance(w, Gtk.CheckButton):
+            label = w.get_child()
+            label.set_line_wrap(True)
+        elif isinstance(w, Gtk.Container):
+            w.foreach(self._wrap_checkbox_labels)
+
+    def inject_extensions_preferences(self, init=False):
+        if not init:
+            # remove preferences buttons for all extensions
+            assert(self.prefs_stack.get_visible_child_name() == 'extensions')
+            found_extensions = False
+            for child in self.prefs_stack.get_children():
+                if found_extensions:
+                    self.prefs_stack.remove(child)
+                elif child == self.prefs_stack.get_visible_child():
+                    found_extensions = True
+
+        # add preferences buttons for all extensions
         result = gpodder.user_extensions.on_preferences()
         if result:
             for label, callback in result:
                 self.prefs_stack.add_titled(callback(), label, label)
```

Comment:
This line would then become `self.prefs_stack.add_titled(callback(), "extension." + label, label)`